UKHomeOffice/nmsw-ui
# NMSW-1098 Show list of passengers

# Ticket

NMSW-1098

----

## AC

need to make API call to get people
need to make container page to hold this and be ready for the other elements
need to map api results into a component and display all

----

## To test

- create report
- upload good FAL 6 passengers file
- click continue option
- > should see list of passengers that you uploaded


----

## Notes
